test/es-module/test-esm-experimental-warnings.mjs

```diff
@@ -1,10 +1,33 @@
 import { mustCall } from '../common/index.mjs';
 import { fileURL } from '../common/fixtures.mjs';
-import { match, strictEqual } from 'assert';
+import { doesNotMatch, match, strictEqual } from 'assert';
 import { spawn } from 'child_process';
 import { execPath } from 'process';
 
-// Verify experimental warnings are printed
+// Verify no warnings are printed when no experimental features are enabled
```

Comment:
```suggestion
// Verify no warnings are printed when no experimental features are enabled or used
```

Some experimental warnings are printed irrespective of flags, like if someone uses the new global `fetch`, which doesn’t require a flag.